lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +19,21 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if isinstance(text, str):
+        text_without_punctuation = ""
+        for i in text:
+            if i.isalpha():
```

Comment:
This is not a check 'is not punctuation' - this check about 'is alpha and is digit and is a space'

Reply:
Done